fs/filter.go

```diff
@@ -23,6 +24,8 @@ var (
 	includeRule    = pflag.StringP("include", "", "", "Include files matching pattern")
 	includeFrom    = pflag.StringP("include-from", "", "", "Read include patterns from file")
 	filesFrom      = pflag.StringP("files-from", "", "", "Read list of source-file names from file")
+	minAge         = pflag.Int64P("min-age", "", 0, "Don't transfer any file younger than N seconds")
```

Comment:
I would suggest you use a `time.Duration`, see [DurationP](https://godoc.org/github.com/spf13/pflag#DurationP). This makes it much easier to specify minutes, hours, etc.